site/docs/master/csi.md

```diff
@@ -25,5 +25,28 @@ This section documents some of the choices made during implementation of the Vel
 
 # Roadmap
 
-Velero's support level for CSI volume snapshotting will follow upstream Kubernetes support for it, and will reach general availability sometime
+Velero's support level for CSI volume snapshotting will follow upstream Kubernetes support for the feature, and will reach general availability sometime
 after volume snapshotting is GA in upstream Kubernetes. Beta support is expected to launch in Velero v1.4.
+
+# Enabling CSI Support
```

Comment:
I would include the plugin registration step here as well. I'd probably also move this up to just under "Prerequisites", and since this section would now cover all installation/setup on the Velero side, I'd remove the feature flag and plugin steps from "Prerequisites" (in my mind they're not prereqs, they're the installation process).

Reply:
we have the feature flag and plugin registration in Prereqs 3, 4 may need to link the plugin image.